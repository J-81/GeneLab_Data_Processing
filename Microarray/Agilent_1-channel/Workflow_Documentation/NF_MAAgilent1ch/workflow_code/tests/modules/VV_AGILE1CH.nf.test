nextflow_process {

    name "Test Process VV_AGILE1CH"
    script "modules/VV_AGILE1CH.nf"
    process "VV_AGILE1CH"

    test("Should run without failures") {

        when {
            params {
                // define parameters here. Example:
                // outdir = "tests/results"
            }
            process {
                """
                // define inputs of the process here. Example:
                // input[0] = file("test-file.txt")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
