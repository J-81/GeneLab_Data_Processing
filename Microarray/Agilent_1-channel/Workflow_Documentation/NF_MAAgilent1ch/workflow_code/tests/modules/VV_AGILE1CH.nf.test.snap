{
    "Should run without failures": {
        "content": [
            [
                [
                    "SampleTable.csv:md5,f393473808e2b9e262c7da5809e0baca",
                    "contrasts.csv:md5,c9af9056cef7bd0210a626cd9c6175b2",
                    "differential_expression.csv:md5,7aabea95a4b1d09ec26fb826daa17f08",
                    "visualization_PCA_table.csv:md5,6b37103ca8cacdfc2a924631e7422fa9",
                    "visualization_output_table.csv:md5,6cac37993daf664d1be489b1f5a9befd"
                ]
            ],
            [
                "GLDS-367_microarray_v0_runsheet.csv:md5,6b066905180d502a70e4ad9fc094433d"
            ]
        ],
        "timestamp": "2023-01-30T19:57:11+0000"
    }
}